src/aap_eda/core/models/activation.py

```diff
@@ -74,6 +83,57 @@ class Meta:
     status_updated_at = models.DateTimeField(null=True)
     status_message = models.TextField(null=True, default=None)
 
+    def save(self, *args, **kwargs):
+        # when creating
+        if self._state.adding:
+            if self.status_message is None:
+                self._set_status_message()
+        else:
+            if not bool(kwargs):
```

Comment:
We could call the method with any other param but without the update_fields and it would pass this check. If want to enforce the use of update_field, we must check for its existence.